phpcs.xml

```diff
@@ -72,11 +72,22 @@
     <rule ref="Squiz.Functions.FunctionDuplicateArgument" />
     <rule ref="Squiz.Functions.GlobalFunction" />
     <rule ref="Squiz.Scope" />
+
+
+    <rule ref="Squiz.Strings.DoubleQuoteUsage" />
+    <rule ref="Squiz.Strings.DoubleQuoteUsage.ContainsVar">
```

Comment:
Do you have a specific use case for this in mind? There is none in the current code as far as I can tell.